MapboxNavigation/PollyVoiceController.swift

```diff
@@ -66,17 +68,15 @@ public class PollyVoiceController: RouteVoiceController {
     }
     
     public override func didPassSpokenInstructionPoint(notification: NSNotification) {
```

Comment:
PollyVoiceController should override `speak(_:)`, but should have no reason to override `didPassSpokenInstructionPoint(_:)`. Move everything from this method into `speak(_:)`, except for the statements that are also in `RouteVoiceController.didPassSpokenInstructionPoint(_:)`. Then remove this method override. The implementation in `RouteVoiceController.didPassSpokenInstructionPoint(_:)` will get called on PollyVoiceController whenever we pass a spoken instruction point.